src/Symfony/Bundle/FrameworkBundle/Resources/config/workflow.xml

```diff
@@ -7,24 +7,24 @@
     <services>
         <defaults public="false" />
 
-        <service id="workflow.abstract" class="Symfony\Component\Workflow\Workflow" abstract="true" public="true">
+        <service id=".workflow.abstract" class="Symfony\Component\Workflow\Workflow" abstract="true" public="true">
             <argument /> <!-- workflow definition -->
             <argument type="constant">null</argument> <!-- marking store -->
             <argument type="service" id="event_dispatcher" on-invalid="ignore" />
             <argument /> <!-- name -->
         </service>
-        <service id="state_machine.abstract" class="Symfony\Component\Workflow\StateMachine" abstract="true" public="true">
+        <service id=".state_machine.abstract" class="Symfony\Component\Workflow\StateMachine" abstract="true" public="true">
             <argument /> <!-- workflow definition -->
             <argument type="constant">null</argument> <!-- marking store -->
             <argument type="service" id="event_dispatcher" on-invalid="ignore" />
             <argument /> <!-- name -->
         </service>
 
-        <service id="workflow.marking_store.method" class="Symfony\Component\Workflow\MarkingStore\MethodMarkingStore" abstract="true" />
+        <service id=".workflow.marking_store.method" class="Symfony\Component\Workflow\MarkingStore\MethodMarkingStore" abstract="true" public="false" />
 
         <service id="workflow.registry" class="Symfony\Component\Workflow\Registry" />
         <service id="Symfony\Component\Workflow\Registry" alias="workflow.registry" />
 
-        <service id="workflow.security.expression_language" class="Symfony\Component\Workflow\EventListener\ExpressionLanguage" />
+        <service id=".workflow.security.expression_language" class="Symfony\Component\Workflow\EventListener\ExpressionLanguage" public="false" />
```

Comment:
isn't this already private thanks to the defaults ?

Reply:
You are right :+1: I forgot that :)